docs/advanced.md

```diff
@@ -577,13 +577,11 @@ import httpx
 r = httpx.get("https://example.org", verify="path/to/client.pem")
 ```
 
-Alternatively, you can pass a standard library `ssl.SSLContext`.
+Alternatively, you can use `httpx.create_ssl_context`.
 
 ```python
->>> import ssl
 >>> import httpx
->>> context = ssl.create_default_context()
->>> context.load_verify_locations(cafile="/tmp/client.pem")
```

Comment:
I wouldn't replace this section, instead I think we should leave it as is, but then supplement it by following it with something like this...

---

We also include a helper function for creating properly configured `SSLContext` instances...

```python
>>> context = httpx.create_ssl_context()
```

The `create_ssl_context` function accepts the same set of SSL configuration arguments as `httpx.Client`...

* `verify` - Blah blah blah
* `cert` - Blah blah blah
* `trust_env` - Blah blah blah
* `http2` - Blah blah blah

---

That'd help make it clear that you can either generate the SSLContext instance entirely yourself, or else use the helper function.